tests/dt/test-fillna.py

```diff
@@ -61,6 +68,9 @@ def test_fillna_str():
   assert str(fillna(f.A + f.B, reverse = True)) == "FExpr<" + fillna.__name__ + "(f.A + f.B, reverse=True)>"
   assert str(fillna(f.B, reverse = True)) == "FExpr<" + fillna.__name__ + "(f.B, reverse=True)>"
   assert str(fillna(f[:2], reverse = False)) == "FExpr<"+ fillna.__name__ + "(f[:2], reverse=False)>"
+  assert str(fillna(f.A, value = 2)) == "FExpr<" + fillna.__name__ + "(f.A, value=2)>"
+  assert str(fillna([f.A, f.B], value = 2)) == "FExpr<" + fillna.__name__ + "([f.A, f.B], value=2)>"
+  assert str(fillna([f.A, f.B], value = 2, reverse = True)) == "FExpr<" + fillna.__name__ + "([f.A, f.B], value=2)>"
```

Comment:
Need a test for the case when both parameters (`value` and `reverse`) are provided.